tools/bcdb_mock/models/generated/workloads/workloads.go

```diff
@@ -93,9 +93,20 @@ type TfgridWorkloadsReservationContainer1 struct {
 	Volumes           []TfgridWorkloadsReservationContainerMount1    `bson:"volumes" json:"volumes"`
 	NetworkConnection []TfgridWorkloadsReservationNetworkConnection1 `bson:"network_connection" json:"network_connection"`
 	StatsAggregator   []TfgridWorkloadsReservationStatsaggregator1   `bson:"stats_aggregator" json:"stats_aggregator"`
+	Logs              []TfgridWorkloadsReservationLogs1              `bson:"logs" json:"logs"`
 	FarmerTid         int64                                          `bson:"farmer_tid" json:"farmer_tid"`
 }
 
+type TfgridWorkloadsReservationLogs1 struct {
```

Comment:
Can we add the corresponding schema structures to the files under `models/schema/workloads`. No regeneration is needed, just to be able to port this back to jumpscale.

Reply:
Please review my changes to see if models seems right :)